crates/wdl-engine/src/eval/v1/task.rs

```diff
@@ -1626,7 +1621,7 @@ impl TaskEvaluator {
         inputs: &TaskInputs,
         attempt: u64,
         previous_requirements: Option<Arc<HashMap<String, Value>>>,
-    ) -> EvaluationResult<EvaluatedSections> {
+    ) -> EvaluationResult<(EvaluatedSections, u64)> {
```

Comment:
I don't love this, but I didn't love mixing it in with the very clean `EvaluationSections` struct either. The alternative is calculating the field twice, which I think is just a recipe for a bug.

Reply:
Alright, will do.